Test/dafny0/ReadsOnMethods-Resolution.dfy.expect

```diff
@@ -0,0 +1,10 @@
+ReadsOnMethods-Resolution.dfy(7,28): Error: reads clauses on methods are forbidden without the --reads-clauses-on-methods option
```

Comment:
Oh I thought it would be  parser warning. Forget my other comment then. An error during resolution is fine.
```suggestion
ReadsOnMethods-Resolution.dfy(7,28): Error: reads clauses on methods are forbidden without the command-line flag `--reads-clauses-on-methods`
```